package.json

```diff
@@ -60,13 +60,15 @@
     "@reach/tabs": "^0.16.1",
     "@reach/visually-hidden": "^0.16.0",
     "classnames": "^2.3.1",
+    "mdast": "^3.0.0",
     "next": "^11.1.2",
     "next-mdx-remote": "^3.0.5",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-intersection-observer": "^8.32.1",
     "slugify": "^1.6.4",
-    "swingset": "^0.10.2"
+    "swingset": "^0.10.2",
+    "unist-util-visit": "^4.1.0"
```

Comment:
I've added `unist-util-visit` to create a `remark` plugin to fix the `/api` links in Vault content, and have added `mdast` for clearer typing of that plugin.